crates/tauri/src/ipc/channel.rs

```diff
@@ -149,15 +151,14 @@ impl JavaScriptChannelId {
         match body {
           // Don't go through the fetch process if the payload is small
           InvokeResponseBody::Json(string) if string.len() < MAX_JSON_DIRECT_EXECUTE_THRESHOLD => {
-            webview.eval(format!(
-              "window['_{callback_id}']({{ message: {string}, index: {current_index} }})"
-            ))?;
+            webview.eval(format_callback::format_raw(
+              CallbackFn(callback_id),
+              format!("{{ message: {string}, index: {current_index} }}"),
+            )?)?;
           }
           InvokeResponseBody::Raw(bytes) if bytes.len() < MAX_RAW_DIRECT_EXECUTE_THRESHOLD => {
             let bytes_as_json_array = serde_json::to_string(&bytes)?;
-            webview.eval(format!(
-              "window['_{callback_id}']({{ message: new Uint8Array({bytes_as_json_array}).buffer, index: {current_index} }})",
-            ))?;
+            webview.eval(format_callback::format_raw(CallbackFn(callback_id), format!("{{ message: new Uint8Array({bytes_as_json_array}).buffer, index: {current_index} }}"))?)?;
```

Comment:
I think `format_callback::format_raw` is not supposed to be dealing with js code but just plain json, and `new Uint8Array` might not work here?

Let's add a new function and use it instead:

```rust
pub fn format_raw_js(id: u32, js: &str) -> crate::Result<String> {
  format!(
  r#"
  if (window["_{id}"]) {{
    window["_{id}"]({arg})
  }} else {{
    console.warn("[TAURI] Couldn't find callback id {id} in window. This happens when the app is reloaded while Rust is running an asynchronous operation.")
  }}"#,
  )
}
```

Also we can change `format_raw`'s callback to use it as well